src/main/java/com/rocketden/main/model/TestCase.java

```diff
@@ -0,0 +1,23 @@
+package com.rocketden.main.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Getter
+@Setter
+public class TestCase {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+
+    private String input;
+    private String expected;
+    private Boolean hidden = false;
```

Comment:
Out of curiosity do you know what this would be used for? If you're thinking like how LeetCode usually provides a few examples with explanations, I would have imagined that would be included in the project description since it would only be for the first 1-2 of 10+ test cases, but I may be wrong on that - lmk

Reply:
Yeah, I was thinking the LeetCode example, and this could be particularly useful if we did end up choosing a more Kahoot-style system, meaning that the teachers would definitely want to explain complex test cases to students in as easy a method as possible.

Reply:
Sounds good - in that case, if it's cool I'll add a card to the Trello so we can revisit it later. Mainly holding off on this for now cause I think there are a couple ways of implementing that behavior, which we can also talk about in my next PR when I implement some of these features